magento/magento2
# [Fixed #26302] Product grid has quantity in decimals #26302

### Description (*)

Product grid has quantity in decimals #26302

### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

Product grid has quantity in decimals #26302

### Manual testing scenarios (*)

1. Add product
2. Go to product Grid
3. Check qty value in Grid

### Questions or comments


### Contribution checklist (*)
